plugins/hydra_nevergrad_sweeper/hydra_plugins/hydra_nevergrad_sweeper/core.py

```diff
@@ -96,29 +96,54 @@ def parse(cls, override: Override) -> Any:
 
 
 def get_nevergrad_params(override: Override) -> Any:
-
     """
+     # regardless of the choice values, choice is always unordered p.Choice:
+     choice(a,b,c)  =>  ng.p.Choice(["a", "b", "c"])
 
-   choice(1,2,3)  ==>  ng.p.Choice([["Helium", "Nitrogen", "Oxygen"]])
-   tag(transition,choice(2,3,4))  ==> ng.p.TransitionChoice(["Solid", "Liquid", "Gas"])
-   tag(transition, range(1, 11, 2)) ==> ng.p.TransitionChoice([1, 3, 5 ...])
-   range(1,5) ==>  ng.p.Scalar(lower=1, upper=12).set_integer_casting(),
-   interval(1.0,5.0) ==> np.p.Scalar(lower=1.0, upper=5.0)
-   tag(log, interval(0.1, 1)) ==> np.p.Log(lower=0.1, upper=1)
-    """
+     # We can support forcing a choice to be ordered by tagging it:
+     # (I prefer order over transition(al). at least to me it's clearer.
```

Comment:
No need to mention your preferences in the comments. also order vs ordered.
Once the decision is made, it is what it is. no need to include a full history of every decision in the docs.

Reply:
This is outdated code, something wrong with github UI?